webpack.common.js

```diff
@@ -1,20 +1,21 @@
+/* eslint-disable @typescript-eslint/no-var-requires -- minimising changes */
 const path = require('path');
-const { merge } = require('webpack-merge');
 const AssetsPlugin = require('assets-webpack-plugin');
-const webpack = require('webpack');
+const babelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
 const GitRevisionPlugin = require('git-revision-webpack-plugin');
 const NodePolyfillPlugin = require('node-polyfill-webpack-plugin');
-const babelLoaderExcludeNodeModulesExcept = require('babel-loader-exclude-node-modules-except');
+const webpack = require('webpack');
+const { merge } = require('webpack-merge');
 const nodeExternals = require('webpack-node-externals');
 
 const assetsPluginInstance = new AssetsPlugin({
 	path: path.resolve(__dirname, './dist/'),
 });
 
 const definePlugin = new webpack.DefinePlugin({
-	WEBPACK_BUILD: process.env.TEAMCITY_BUILD
-		? `'${process.env.TEAMCITY_BUILD}'`
+	WEBPACK_BUILD: process.env.GITHUB_RUN_NUMBER
+		? `'${process.env.GITHUB_RUN_NUMBER}'`
 		: `'DEV_${new Date().getTime()}'`,
 	GIT_COMMIT_HASH: process.env.BUILD_VCS_NUMBER
```

Comment:
what is `BUILD_VCS_NUMBER`? is that also a teamcity thing

Reply:
Yeah, that's another TeamCity environment variable, although it has a fallback. The commit hash is used by prout to determine when a change is live: https://manage.theguardian.com/_prout

[`GITHUB_SHA`](https://docs.github.com/en/actions/learn-github-actions/variables) looks like a suitable replacement?

Reply:
Good catch, I just assumed it was something else entirely. `GITHUB_SHA` looks perfect!